src/function/special/riemannZeta.js

```diff
@@ -0,0 +1,140 @@
+import { factory } from '../../utils/factory.js'
+
+const name = 'zeta'
+const dependencies = ['typed', 'config', 'multiply', 'pow', 'divide', 'factorial', 'equal', 'gamma', 'sin', 'subtract', 'add', 'Complex', 'BigNumber']
+
+export const createRiemannZeta = /* #__PURE__ */ factory(name, dependencies, ({ typed, config, multiply, pow, divide, factorial, equal, gamma, sin, subtract, add, Complex, BigNumber }) => {
+  /**
+   * Compute the Riemann Zeta function of a value using an infinite series for
+   * all of the complex plane using Riemann's Functional equation.
+   *
+   * Based off the paper by Xavier Gourdon and Pascal Sebah
+   * ( http://numbers.computation.free.fr/Constants/Miscellaneous/zetaevaluations.pdf )
+   *
+   * Implementation and slight modification by Anik Patel
+   *
+   *
+   * Syntax:
+   *
+   *    math.zeta(n)
+   *
+   * Examples:
+   *
+   *    math.zeta(5)       // returns 1.0369277551433699...
+   *    math.zeta(-0.5)    // returns -0.2078862249773...
+   *    math.zeta(math.i)  // returns 0.0033002..., -0.4181554491...i
+   *
+   *
+   * @param {number | Complex | BigNumber} s   A real or complex number
```

Comment:
What would be a more informative description for the property `s` than "A real or complex number"? (missing BigNumber right now)

Reply:
😂  well it's technically OK now. What I meant is that the description is not really a description, it only tells the type of `s`, but that is already documented. 

It's like when you ask me "what do you have there?" and my answer is "steel and plastic". That can be technically correct. But a more helpful answer would be "a screwdriver". Explaining what kind of thing it is or what meaning it has.

Not a big deal though. I can't come up with something meaningful. Maybe "The numeric input for the zeta function" or so.

Reply:
Oh I see what you are saying, sorry for the misunderstanding. I think I can come up something more descriptive